contrib/packs/actions/pack_mgmt/info.py

```diff
@@ -13,19 +13,43 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from st2actions.runners.pythonrunner import Action
-import json
 import os
+import json
+
+from st2actions.runners.pythonrunner import Action
+from st2common.content.utils import get_packs_base_paths
 
 GITINFO_FILE = '.gitinfo'
 
 
 class PackInfo(Action):
-    def run(self, pack, pack_dir="/opt/stackstorm/packs"):
-        gitinfo = os.path.join(pack_dir, pack, GITINFO_FILE)
+    def run(self, pack):
+        packs_base_paths = get_packs_base_paths()
+
+        pack_git_info_path = None
+        for packs_base_path in packs_base_paths:
+            git_info_path = os.path.join(packs_base_path, pack, GITINFO_FILE)
+
+            if os.path.isfile(git_info_path):
+                pack_git_info_path = git_info_path
+                break
+
+        error = ('Pack "%s" doesn\'t exist or it doesn\'t contain a valid .gitinfo file' % (pack))
```

Comment:
Heh, yeah.

I originally re-used the same error message in both places, but later on I decided to make the second error message a bit more descriptive, but I forgot to move the variable declaration.